src/reanimated2/animation/util.ts

```diff
@@ -73,6 +75,15 @@ function recognizePrefixSuffix(value: string | number): RecognizedPrefixSuffix {
   }
 }
 
+export function shouldReduceMotion(config?: ReducedMotionConfig) {
```

Comment:
As discussed offline, it took me a while to understand what this function exactly does. Also, throughout the examples we often use `shouldReduceMotion` as a boolean. Can we think of a better name for this function?
```suggestion
export function shouldReduceMotionForConfig(config?: ReducedMotionConfig) {
```